src/components/CytoscapeGraph/CytoscapeGraphUtils.ts

```diff
@@ -88,3 +88,11 @@ export const runLayout = (cy: any, layout: Layout) => {
 
   cy.scratch(CytoscapeGlobalScratchNamespace).showNodeLabels = showNodeLabels;
 };
+
+export const edgeData = (ele: any): DecoratedGraphEdgeData => {
```

Comment:
Since this PR is about type safety: maybe give `ele` a type. 
Trivial though, if you don't want to that is also fine.

Reply:
`unknown` might be better suited here, or `<T extends { data: () -> unknown }>`.

Reply:
Thinking a little more about it:
* The `DecoratedGraph*Data` types seem equal, even though some of the names are different. Maybe they should be generalized in a further PR?
* Since they are interfaces, we can check the "capabilities", like `has http` or `has grpc`, and then we can just receive the capabilities as extends on the generics.

Reply:
Yes, more work can be made in that direction. I think that currently we need a way to ensure if what we are receiving (from the server) is really what we are expecting, else this added type-safety will be just an illusion.